test/babel/fixtures/missingImport.js

```diff
@@ -0,0 +1,5 @@
+/* @jsx createElement */
+import { createElement } from 'style-sheet'
```

Comment:
When creating the library import `import { StyleSheet } from 'style-sheet'` this is lost. We need to merge them instead i.e. push `StyleSheet` to the `ImportDeclaration.specifiers` array